### PR TITLE
Everything seems to work 

### PR DESCRIPTION
**Pull request by natsukagami**
image
image

### CODE REVIEW

**Review by ann-kilzer — COMMENTED** [review]
Looking good, a couple nits -- don't need to change these for the hackathon but good to consider if we revisit some of this code for a future SDK.

**Review by ann-kilzer — APPROVED** [review]
LGTM :shipit: 🚢 🚀 

Great!!!!

As a future enhancement we could figure out what to do with the button on mobile screens, but let's get this in.

want-nintendo-switch

File: web/src/plugins/cgutils.js [hunk 1]
```diff
@@ -16,37 +16,48 @@ export default {
       // method: the name of the read method
       // sender: the from address for the transaction
       // args: the arguments for the method call
-      async callMethod(contract, instance, method, sender, apiKey, args) {
+      async callMethod(contract, instance, method, sender, apiKey, args, data) {
```
[the remark this answers is not in the document]
Comment (natsukagami) [reply web/src/plugins/cgutils.js:19]: It's a [janky](https://youtu.be/S-3wEC6Fj_8?t=3404) fix! I wanted to include `contractOverride: true` into part of the request's JSON body but the current function did have it, so I added the `data` field that is going to be merged into the request's body.

File: web/src/plugins/cgutils.js [hunk 2]
```diff
@@ -16,37 +16,48 @@ export default {
...
       },
-      // Sends an axios request to multibaas
-      async methodPostHelper(contract, instance, method, sender, apiKey, args) {
+      async get(url, apiKey) {
+        return axios.get(`${BASE_URL}${url}`, {
+          credentials: 'same-origin',
+          headers: { Authorization: `Bearer ${apiKey}` },
+        });
+      },
+      async post(url, apiKey, data) {
```
[the remark this answers is not in the document]
Comment (natsukagami) [reply web/src/plugins/cgutils.js:35]: `post` is just a generic `POST` request builder, with the URL automatically prefixed with our BASE_URL.
`methodPostHelper` is used for specifically calling a contract method.

**Pull request merged**